gridstatus/caiso.py

```diff
@@ -72,16 +71,22 @@ def get_fuel_mix(self, date, end=None, verbose=False):
         """Get fuel mix in 5 minute intervals for a provided day
 
         Arguments:
-            date (datetime or str): "latest", "today", or an object that can be parsed as a datetime for the day to return data.
+            date (datetime or str): "latest", "today", or an object
```

Comment:
something weird is happening with how this is showing up in the docs

here is what it looks like [currently](https://docs.gridstatus.io/en/latest/autoapi/gridstatus/index.html#gridstatus.CAISO.get_fuel_mix):

<img width="800" alt="image" src="https://user-images.githubusercontent.com/371792/210280187-f8021ed3-4a17-4a73-bf6e-6c7608e9a7f8.png">


and here is what it looks like from [this PR](https://isodata--114.org.readthedocs.build/en/114/autoapi/gridstatus/index.html#gridstatus.CAISO.get_fuel_mix)
<img width="800" alt="image" src="https://user-images.githubusercontent.com/371792/210280163-9251d427-477b-411b-936a-6826af4f798f.png">

Reply:
i think the ideal case would be for just the argument and type to be bolded. if not possible, no bold

Reply:
looking much nicer now :)

Reply:
Fixed by adding napoleon extension to Sphinx and following [Google Style Python Docstrings format](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html)